pages/content/pixi/index.md

```diff
@@ -0,0 +1,15 @@
+---
+$sitemap:
+  enabled: false
+globalText:
```

Comment:
Can we find a better name than `globalText`? It's not clear to me what this means.

Reply:
As discussed offsite its now "staticText"